site/content/authors/default.mdx

```diff
@@ -0,0 +1,5 @@
+---
+name: default
+avatar: /assets/images/avatar.png
+type: Author
+---
```

Comment:
You can define default values for contentlayer document type fields, like we do e.g. here `layout: { type: "string", default: "blog" }` 😜